### DISCUSSION TITLE
I keep getting this error about Xformers

### DISCUSSION THREAD

**Discussion by ItsZourk** [TheLastBen/fast-stable-diffusion#2687] [General]
I was using the colab version with no problem, and all of a sudden i started getting this error after restarting a session. I am using the Colab Pro.
![Screenshot_3](https://github.com/TheLastBen/fast-stable-diffusion/assets/153991313/87ccef74-94e5-4138-b1f8-4d5a8bc1af74)

I tried fresh install on google drive, tried to update xformers, nothing worked so far.

**Comment by ClaraSanders** [comment]
This is due to a Google update. Many other colabs are crashing if not all. We'll have to wait.

**Comment by Sttasha** [comment]
same here 0____0

**Comment by wanote26** [comment]
Somebody gave a solution that seemed to work for the same error a while ago, here:
https://github.com/TheLastBen/fast-stable-diffusion/issues/2681#issuecomment-1858714824

**Reply by wanote26** [reply]
I just tried it and it's working for me as of now

**Reply by ClaraSanders** [reply]
Yeah, it works now. Anyway, I will wait for Ben's update.
